example/basic/package.json

```diff
@@ -6,17 +6,11 @@
     "react-scripts": "0.2.1"
   },
   "dependencies": {
-    "derivable": "^1.0.0-beta10",
-    "react": "^15.2.1",
-    "react-derivable": "../../",
-    "react-dom": "^15.2.1"
+    "react-derivable": "link:../.."
```

Comment:
Wow, is it realtime update? Nice!

Reply:
Not quite as `src/` should be built into `lib/`.

Reply:
Still better then reinstalling